source/apis/routes.py

```diff
@@ -1,13 +1,32 @@
 from pathlib import Path
 
-from fastapi import APIRouter, Response
+import fitz
+from fastapi import APIRouter, Response, Query
 
-from source.apis.utils import generate_pdf, get_available
+from source.services.utils import Aradocgen
 
 router = APIRouter()
 
 text = Path(r"C:\Users\skand\PycharmProjects\DocGeneratorFastApi\source\models\data.txt").read_text(encoding="utf-8")
 
+fonts = \
```

Comment:
For the fonts, we need to keep them inside the root directory and so there is no need for the full path, use the relative path, for example: DocGeneratorFastApi/fonts would be a good directory!